pyproject.toml

```diff
@@ -1,3 +1,96 @@
-[build-system]
-requires = ["setuptools"]
-build-backend = "setuptools.build_meta"
```

Comment:
We shouldn't remove this. Setuptools is assumed but that's far too esoteric to ask people to decipher. 


https://packaging.python.org/en/latest/specifications/pyproject-toml/#declaring-build-system-dependencies-the-build-system-table

Reply:
Not sure what you mean here, as uv build doesn't need this, but I put it back in and doesn't seem to pose any issue.